translations/ui-users/ca.json

```diff
@@ -609,7 +609,7 @@
     "reports.overdue.item.callNumberComponents.suffix": "Call number suffix",
     "reports.overdue.item.enumeration": "Enumeration",
     "reports.overdue.item.volume": "Volume",
-    "loans.details.effectiveCallNumber": "Effective call number string",
+    "loans.details.effectiveCallNumber": "Effective call number",
```

Comment:
Please update only `en.json` and let the change there feed the other locale files. Any changes made here will just be overwritten the next time the import-from-lokalise script runs. 

(That said, I am guilty of doing this too when a PO reports an error in a translation string because it's the only way I can cause the fix to be visible to a PO, who is the only one with the power to close a ticket. @dltj, is there/could there be a way for devs to have some control of this process, or do you think the Right Fix is simply to educate all parties about how long these changes take to filter through the system -- is there documentation about how it works (is it periodic, automatic based on the presence of changes, manual, etc)?)

Reply:
I'd like to totally automate the handling of translations ([FOLIO-1820|https://issues.folio.org/browse/FOLIO-1820]) to be a nighly thing, but the lack of stability in the test means that I have to go in and restart the Jenkins jobs for at least a couple repositories every time I run the lokalise batch job.  I'm hoping the new testing framework brings that stability!

Reply:
I should add that it is okay to update translation files other than `en.json`—just be aware that they will be overwritten the next time that the translations from Lokalise are imported.